app/backup.py

```diff
@@ -481,7 +490,13 @@ def _get_dest_dir(self, c: Container, src_dir_name: str) -> Tuple[Path, str]:
 
     def _format_dated_folder(self, base_dest_dir: Path, folder: str) -> Path:
         """Format the destination folder with the date"""
-        time_format = str(time.strftime(self.env.DEST_DATE_FORMAT))
+
+        # Use Nautical start time as the date for formatting
+        time_format = str(self.start_time.strftime(self.env.DEST_DATE_FORMAT))
+        if self.env.USE_CONTAINER_BACKUP_DATE:
+            # Use current time for formatting
+            self.log_this(f"Using current time for date formatting since DEST_DATE_FORMAT=true", "Trace")
```

Comment:
I also guess this is an error on the logging message:
```
DEST_DATE_FORMAT=true => USE_CONTAINER_BACKUP_DATE=true
```
There is a simlar error on line: 476

Reply:
100%, this log was mislabeled